app/scripts/modules/ecs/ecs.help.ts

```diff
@@ -0,0 +1,24 @@
+import { module } from 'angular';
+import { HELP_CONTENTS_REGISTRY, HelpContentsRegistry } from 'core/help/helpContents.registry';
+
+const helpContents: {[key: string]: string} = {
+  'ecs.loadBalancer.targetGroup': '<p>A <em>target group</em> is attached to an application / network load balancer and are targets for load balancer traffic.</p>  <p> You need to create these resources outside of Spinnaker.</p>',
+  'ecs.serverGroup.clusterName': '<p>The name of the ECS cluster (underlying hosts) onto which your application will be deployed.</p><p>Note that suggestions here are dependent on the selected account and region combination.</p>',
```

Comment:
`cluster` is singular, `hosts` is plural in the first sentence. Probably change to `host`?